tests/integrations/backup/mod.rs

```diff
@@ -416,10 +422,132 @@ fn test_backup_rawkv() {
 }
 
 #[test]
-fn test_backup_raw_meta() {
-    let suite = TestSuite::new(3, 144 * 1024 * 1024);
+fn test_backup_rawkv_v1() {
```

Comment:
Suggest combine the four unit tests into one.

Reply:
if conbined into one, when it fails, it's not intuitive to know which case fails.

Reply:
Failure of unit test is rare. And the stack backtrace can help us find the case.
It would be better to be less verbose.

Reply:
done